tox.ini

```diff
@@ -117,8 +117,8 @@ envlist =
     ;temporarily disabling tests on flask master
     ; python-framework_flask-{py37,py38,py39,py310,pypy3}-flask{latest,master},
     python-framework_flask-{py37,py38,py39,py310,pypy3}-flask{latest},
-    python-framework_graphene-{py27,py36,py37,py38,py39,py310,pypy,pypy3}-graphenelatest,
-    python-framework_graphene-py37-graphene{0200,0201},
+    python-framework_graphene-{py36,py37,py38,py39,py310}-graphenelatest,
```

Comment:
Was there an incompatibility issue for pypy, pypy3, and py27 with the latest version of graphene? We probably want to test some version of pypy and Python 2.7 with newer graphene versions to check for breaking changes (if possible).

Reply:
It seems that Graphene 3 only supports python 3.6, 3.7, and 3.8 (but also works for 3.9 and 3.10)

Reply:
But Graphene 2 still works/is supported with the newer versions of python, so I've added those into the Graphene 2 section of the tox file